src/test/enginesynctest.cpp

```diff
@@ -1445,6 +1445,23 @@ TEST_F(EngineSyncTest, ZeroBPMRateAdjustIgnored) {
             ControlObject::getControl(ConfigKey(m_sGroup2, "rate"))->get());
 }
 
+TEST_F(EngineSyncTest, DISABLED_BeatDistanceBeforeStart) {
```

Comment:
Are you planning to fix this in this PR?

Reply:
no -- this is a much more complex issue and I'm not even sure what's going wrong.  This is a pretty rare circumstance, and shouldn't be a release-blocker, let alone a PR-blocker.

Reply:
https://bugs.launchpad.net/mixxx/+bug/1930143